src/types.ts

```diff
@@ -21,6 +23,7 @@ export interface UploaderArgs {
   url?: string // Change the upload host (Enterprise use)
   clean?: string // Move discovered coverage reports to the trash
   feature?: string // Toggle features
+  upstream: string // Upstream proxy to connect to
```

Comment:
Can we make this optional so we don't have to default it to an empty string everywhere else?